diff --git a/.scripts/run_docker_build.sh b/.scripts/run_docker_build.sh
index c4079b7..16f305a 100755
--- a/.scripts/run_docker_build.sh
+++ b/.scripts/run_docker_build.sh
@@ -93,11 +93,10 @@ docker run ${DOCKER_RUN_ARGS} \
            -e BUILD_OUTPUT_ID \
            -e BINSTAR_TOKEN \
            "${DOCKER_IMAGE}" \
-           bash \
-           "/home/conda/feedstock_root/${PROVIDER_DIR}/build_steps.sh"
+           bash
 
 # verify that the end of the script was reached
 test -f "$DONE_CANARY"
 
 # This closes the last group opened in `build_steps.sh`
-( endgroup "Final checks" ) 2> /dev/null
\ No newline at end of file
+( endgroup "Final checks" ) 2> /dev/null
